chili-epfl/FROG
# Toast notifications on sign up/login

This PR adds toast notifications on success/error while signing up or logging in to FROG. 

**Testing instructions**
Create a new account. If it is created successfully then you will see a success toast. If there are errors an error toast will be displayed which should disappear on closing the modal. Same for logging in 

**Asana task**
https://app.asana.com/0/1129712605359574/1132009327400800/f
